packages/flutter_tools/lib/src/bundle_builder.dart

```diff
@@ -184,7 +185,8 @@ Future<void> writeBundle(
               doCopy = !await shaderCompiler.compileShader(
                 input: input,
                 outputPath: file.path,
-                target: ShaderTarget.sksl, // TODO(zanderso): configure impeller target when enabled.
+                target: ShaderTarget.sksl,
```

Comment:
Is this only used for flutter_tester where using Impeller isn't possible?

Reply:
Oh woops. I think I got this backwards. If Impeller is enabled we need to provide it here. That doesn't solve the -d all case but it does simplify the enable impeller flag

Reply:
Added back todo